src/service/url-replacements-impl.js

```diff
@@ -812,6 +813,30 @@ export class UrlReplacements {
   }
 
   /**
+   * Append URL parameters from data-amp-addparams to URL
+   * @param {string} url
+   * @param {!Element} element An anchor element.
+   * @return {string} url
+   * @private
+   */
+  extendURLParams_(url, element) {
```

Comment:
As discussed with you on slack `opt_bindings` can be used for the replace use case only.